### DISCUSSION TITLE
Cannot find module '@babel/preset-typescript' from '/serverless'

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1468] [Q&A] [answered]
# Bug Report

<img width="1905" alt="image" src="https://user-images.githubusercontent.com/5801015/202610536-df6bde76-7702-424b-916d-fa747943b937.png">

```
index.js:10 Uncaught Error: Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/preset-typescript' from '/serverless'
```

**Comment by denolfe** [comment]
Could you please fill out the rest of the bug template? This isn't enough information for us to look into.

**Comment by jmikrut** [comment]
Hey @Stupidism — it looks like this preset is somehow working its way into your client-side JS bundle, which should not happen. You would never want `@babel/preset-typescript` in your admin bundle. So take a peek through your code and try and find where you are bringing in server-side code, then use Webpack aliases to fix.

Does that make sense? Gonna convert this to a discussion for now!

**Comment by Stupidism** [comment, marked as answer]
I solved it by upgrading @babel/core and @babel/preset-typescript in my dependencies
